index.bs

```diff
@@ -1444,6 +1446,34 @@ with parameter |value| is:
 
 </div>
 
+<pre class=simpledef>
+Capability: <dfn export>intercept file dialogs</dfn>
+Key: "<code>interceptFileDialogs</code>"
+Value type: boolean
+Description: Defines whether the file dialogs should be intercepted and not shown on screen.
+</pre>
+
+<div algorithm="interceptFileDialogs capability deserialization algorithm">
+The [=additional capability deserialization algorithm=] for the
+"<code>interceptFileDialogs</code>" capability, with parameter |value| is:
+
+1. If |value| is not a boolean, return [=error=] with [=error code|code=]
+   [=invalid argument=].
+
+1. Return [=success=] with data |value|.
+
+</div>
+
+<div algorithm="interceptFileDialogs capability serialization algorithm">
+The [=matched capability serialization algorithm=] for the "<code>interceptFileDialogs</code>" capability,
+with parameter |value| is:
+
+1. If |value| is false, return [=success=] with data null.
+
+1. Return [=success=] with data true.
```

Comment:
```suggestion
1. Return [=success=] with data false.
```

Reply:
This algo is about matching capabilities, not the default value.